app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -1,55 +1,50 @@
-import { h, Component } from 'preact';
+import { h } from 'preact';
 import PropTypes from 'prop-types';
-
+import { useEffect, useState } from 'preact/hooks';
 import { userData, getContentOfToken, updateOnboarding } from '../utilities';
 import Navigation from './Navigation';
+import OnboardingForm from './OnboardingForm';
+import OnboardingContentHeader from './OnboardingContentHeader';
 
-/* eslint-disable camelcase */
-class ProfileForm extends Component {
-  constructor(props) {
-    super(props);
-
-    this.handleChange = this.handleChange.bind(this);
-    this.onSubmit = this.onSubmit.bind(this);
-    this.user = userData();
+const lastOnboardingPage = 'v2: personal info form';
 
-    this.state = {
-      formValues: {
-        summary: '',
-        location: '',
-        employment_title: '',
-        employer_name: '',
-      },
-      last_onboarding_page: 'v2: personal info form',
-      canSkip: true,
-    };
-  }
+const ProfileForm = ({
```

Comment:
API documentation (blocking): For all exportable code, please include JSDoc comments. An example of this can be found at https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js

Since we need to document this as a function, it's preferable to use a function instead of an arrow function.

```suggestion
function ProfileForm = ({
```